src/features/nft/reviewModal/index.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
```

Comment:
@muhadovic change the file name

Reply:
@dbizhev I follow the title of the design for filename. It it is wrong, please suggest the filename